Calling/BlobStorageHelper.cs

```diff
@@ -93,5 +100,32 @@ public static async Task<BlobStorageHelperInfo> UploadFileAsync(string connectio
                 throw new Exception($"The file upload was not successful. Exception: {ex.Message}");
             }
         }
+
+        
+        /// <summary>
+        /// Returns a URI containing a SAS for the blob.
+        /// </summary>
+        /// <param name="containerName">A string containing the name of a container</param>
+        /// <param name="blobClient">A reference to the blobclient object.</param>
+        /// <returns>A string containing the URI for the blob, with the SAS token appended.</returns>
+        static string GetBlobSasUri(string containerName, BlobClient blobClient)
+        {
+            // Create a SAS token that's valid for a given time.
+            BlobSasBuilder sasBuilder = new BlobSasBuilder()
+            {
+                BlobContainerName = containerName,
+                BlobName = blobClient.Name,
+                Resource = "b"
```

Comment:
Can this be anything? I want to make sure we are setting our customers up for success and if we just set a random string it seems confusing?

Reply:
No, it's not anything. If the shared resource is of type blob, we should give resource attribute value as "b". Reference https://docs.microsoft.com/en-us/dotnet/api/azure.storage.sas.blobsasbuilder.resource?view=azure-dotnet#Azure_Storage_Sas_BlobSasBuilder_Resource